### DISCUSSION TITLE
Lost Manager in Distributed Deploment

### DISCUSSION THREAD

**Discussion by samuel809** [Security-Onion-Solutions/securityonion#8521] [General]
What would be the shortest and or more efficient path to recover from an scenario where the manger node is lost?

Distributed so 2.2.130 deployment 
manager + 2 search nodes + 2 fn sensors (1 for host (wazuh and osquery agents) - the other for network traffic)
1 idh

most interested in recovering kibana dashboards.

manager lost due to shutdown, no grub after restart.

**Comment by weslambert** [comment]
You would need to have access to the manager instance's filesystem and copy the indices from disk most likely, then import, or pull out the dashboards from another installation. Otherwise, you might consider booting into recovery mode and rebuilding the boot configuration/grub, etc.
